c++/src/wrap/orc-proto-wrapper.cc

```diff
@@ -32,6 +32,7 @@ DIAGNOSTIC_IGNORE("-Wmissing-variable-declarations")
 DIAGNOSTIC_IGNORE("-Wnested-anon-types")
 DIAGNOSTIC_IGNORE("-Wreserved-id-macro")
 DIAGNOSTIC_IGNORE("-Wshorten-64-to-32")
+DIAGNOSTIC_IGNORE("-Wsuggest-destructor-override")
```

Comment:
During https://github.com/apache/orc/issues/1532, I found that this broke Clang 10 due to `unknown warning group`.